include/libtorrent/file_storage.hpp

```diff
@@ -150,13 +150,13 @@ namespace libtorrent
 		// the number of characters in the name. If this is
 		// name_is_owned, name is 0-terminated and owned by this object
 		// (i.e. it should be freed in the destructor). If
-		// the len is not name_is_owned, the name pointer doesn not belong
+		// the len is not name_is_owned, the name pointer does not belong
 		// to this object, and it's not 0-terminated
 		std::uint64_t name_len:12;
-		std::uint64_t pad_file:1;
-		std::uint64_t hidden_attribute:1;
-		std::uint64_t executable_attribute:1;
-		std::uint64_t symlink_attribute:1;
+		bool pad_file:1;
```

Comment:
this will enlarge this type. this was previously two bitfield units backed by two uint64_ts. now it's 3, two uint64_ts and one bool. some compilers may still coalesce them, but some might not. it's not guaranteed